### PR TITLE
Check styles

### PR DESCRIPTION
**Pull request by lfarrell**

### CODE REVIEW

File: checkstyle.xml [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+		The fcrepo based suppression file disables JavaDoc checks in unit test and IT test files.
+		It also disables indentation checking, e.g. 4 spaces for all java files
+	-->
+	<module name="SuppressionFilter">
+		<property name="file" value="checkstyle-suppressions.xml" />
+	</module>
+	
+	<!-- Checks that there are no tab characters ('\t') in the source code. -->
+	<module name="FileTabCharacter">
+		<property name="eachLine" value="false" />
```
Comment (bbpennel) [inline checkstyle.xml]: Change to true.  Add xsl, jsp

File: checkstyle.xml [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+			It prevents Checkstyle from reporting errors from parts of code that were annotated with @SuppressWarnings and using name of the check to be excluded
+			http://checkstyle.sourceforge.net/config_annotation.html#SuppressWarningsHolder
+		-->
+		<module name="SuppressWarningsHolder" />
+		
+		<!--
+			Checks that local variables that never have their values changed are declared final
+			http://checkstyle.sourceforge.net/config_coding.html#FinalLocalVariable
+			-->
+		<module name="FinalLocalVariable" />
```
Comment (bbpennel) [inline checkstyle.xml]: Remove

File: checkstyle.xml [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+		
+		<!--
+			Check that parameters for methods, constructors, and catch blocks are final
+			http://checkstyle.sourceforge.net/config_misc.html#FinalParameters
+		-->
+		<module name="FinalParameters" />
```
Comment (bbpennel) [inline checkstyle.xml]: Remove

File: checkstyle.xml [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+		-->
+		<module name="MultipleVariableDeclarations" />
+	</module>
+	
+	<!-- No Trailing Whitespace, except on lines that only have an asterisk 
+		(e.g. Javadoc comments) -->
+	<module name="RegexpSingleline">
+		<property name="format" value="(?&lt;!\*)\s+$|\*\s\s+$" />
+		<property name="message" value="Trailing whitespace" />
+		<property name="fileExtensions" value="java,css,js,xml" />
```
Comment (bbpennel) [inline checkstyle.xml]: xsl and jsp

File: checkstyle-suppressions.xml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+     "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+     "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+<suppressions>
+	<suppress checks="JavadocMethod" files="src[/\\]test[/\\]java" />
+	<suppress checks="JavadocPackage" files="src[/\\]test[/\\]java" />
+	<suppress checks="JavadocVariable" files="src[/\\]test[/\\]java" />
+	<suppress checks="JavadocType" files="src[/\\]test[/\\]java" />
+	<suppress checks="JavadocStyleCheck" files="src[/\\]test[/\\]java" />
+	<suppress checks="IndentationCheck" files=".*\.java$" />
+	<suppress checks="MultipleVariableDeclarations" files="src[/\\]test[/\\]java" />
```
Comment (bbpennel) [inline checkstyle-suppressions.xml]: Remove

File: checkstyle.xml (hunk 1 above)
Comment (harringj) [inline checkstyle.xml]: done

File: checkstyle.xml (hunk 2 above)
Comment (harringj) [inline checkstyle.xml]: done

File: checkstyle.xml (hunk 4 above)
Comment (harringj) [inline checkstyle.xml]: done

File: checkstyle-suppressions.xml (hunk 5 above)
Comment (harringj) [inline checkstyle-suppressions.xml]: done

**Pull request merged**